htdocs/product/list.php

```diff
@@ -43,7 +43,9 @@
 require_once DOL_DOCUMENT_ROOT.'/core/class/html.formcompany.class.php';
 require_once DOL_DOCUMENT_ROOT.'/core/class/html.formother.class.php';
 require_once DOL_DOCUMENT_ROOT.'/product/class/html.formproduct.class.php';
-require_once DOL_DOCUMENT_ROOT.'/workstation/class/workstation.class.php';
+if (isModEnabled('workstation')) {
```

Comment:
I don't understand why rhis fix the bug.
The file is included but no code is executed. So even if table is misssing, this should not change nothing ?